python/simulation.py

```diff
@@ -2967,7 +2970,7 @@ def scale_flux_fields(s, flux):
 
 
 def get_flux_freqs(f):
-    return np.linspace(f.freq_min, f.freq_min + f.dfreq * f.Nfreq, num=f.Nfreq, endpoint=False).tolist()
+    return f.freq
```

Comment:
What type does this return?

Reply:
(i.e. what is `type(f.freq)`?)

Reply:
```
>>> type(freqs)
<class 'meep.DoubleVector'>
```

Reply:
Probably want to convert it to a Python list if it is a Meep array.

Reply:
Try
```py
[freqs[i] for i in range(freqs.size())]
```
to make a Python list.